R/calc_mutation_frequency_bin_regions.R

```diff
@@ -0,0 +1,147 @@
+#' @title Mutation counts across sliding windows for multiple regions.
+#'
+#' @description Obtain a long tidy or wide matrix of mutation counts across
+#' sliding windows for multiple regions.
+#'
+#' @details This function takes a metadata table with `these_samples_metadata` 
+#' parameter and internally calls `calc_mutation_frequency_bin_region` 
+#' (that internally calls `get_ssm_by_regions`).
+#' to retrieve mutation counts for sliding windows across one or more regions. 
+#' May optionally provide any combination of a maf data frame, existing metadata,
+#' or a regions data frame or named vector.
+#' The heatmap plotting portion of this function has been moved to 
+#' [GAMBLR.viz::heatmap_mutation_frequency_bin].
```

Comment:
I think it is safe to remove this sentence since the user may not have the GAMBLR.viz installed and this will be a broken link. Or, update this to some generic info not necessarily referring to the vix repo

Reply:
The last sentence has been removed. Thanks!